formatTest/typeCheckedTests/expected_output/reasonComments.re

```diff
@@ -61,36 +73,31 @@ let testingEndOfLineComments = [
   "Item 3", /* Comment For Third Item */
   "Item 4" /* Comment For Fourth Item - but before trailing comma */
   /* Comment after last item in list. */
-]
-/* Comment after rbracket */;
+] /* Comment after rbracket */;
 
 /* The space between ; and comment shoudn't matter */
 let testPlacementOfTrailingComment = [
   "Item 0" /* */
   /* Comment after last item in list. */
-];
+]; /* Comment after semi */
```

Comment:
It's tough for me to recall the approach/algorithm for how this is preserved as an end of line comment, in such a way that the meaning is clear and preserved across reformats.
I know we discussed and you might not have implemented a solution, but documenting it would be a good idea.

```
let x =
   somethingHere (
   ); /* Comment */
```

What is the comment on, the let binding or expression? How is that meaning preserved when there is enough space to fit it on one line:

```
let x = somethingHere ();   /* Comment */
```

Now it is entirely on the let binding, even if we said the original meaning was that the comment applied to the function application only.